internal/apply/apply.go

```diff
@@ -127,3 +128,39 @@ func validateEnvironments(applyEnvironments []string) {
 		}
 	}
 }
+
+func summarizeAll(dir string, projectConfig projectconfig.ZeroProjectConfig) {
+	flog.Infof("Your projects and infrastructure have been successfully created.  Here are some useful links and commands to get you started:")
+
+	graph := projectConfig.GetDAG()
+
+	// Walk the graph of modules and run `make summary`
+	root := []dag.Vertex{projectconfig.GraphRootName}
+	graph.DepthFirstWalk(root, func(v dag.Vertex, depth int) error {
+		// Don't process the root
+		if depth == 0 {
+			return nil
+		}
+
+		name := v.(string)
+		mod := projectConfig.Modules[name]
+		// Add env vars for the makefile
+		envList := []string{
+			fmt.Sprintf("REPOSITORY=%s", mod.Files.Repository),
+		}
+
+		modulePath := module.GetSourceDir(mod.Files.Source)
+		// Passed in `dir` will only be used to find the project path, not the module path,
+		// unless the module path is relative
+		if module.IsLocal(mod.Files.Source) && !filepath.IsAbs(modulePath) {
+			modulePath = filepath.Join(dir, modulePath)
+		}
+
+		envList = util.AppendProjectEnvToCmdEnv(mod.Parameters, envList)
```

Comment:
Should also pass the environments in so the summary can show accurate info based on what was created.